src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -183,6 +183,13 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
             HelpMessage = "A hashtable which represents resource tags.")]
         public Hashtable Tag { get; set; }
 
+        [Parameter(
+            Mandatory = true,
```

Comment:
@akshaysngupta adding a mandatory parameter to an existing cmdlet is a breaking change. You will need to make this optional.